paper.tex

```diff
@@ -856,14 +856,11 @@ \section*{Genome ARGs}
 \caption{\label{fig-arg-in-pedigree}
 % TODO caption needs another pass. One the full section has been written
 % it'll be clearer what we need to say in the caption vs the text.
-The Ancestral Recombination Graph defined as arbitrary inheritance
-of genomes between diploid (or any ploidy; see text) individuals.
-(A) Each individual consists of two genomes, and each genome is
-inherited as a (potentially) recombinant mixture of the individual's parent's
-genomes. (B) Genomes are modelled as nodes in a graph, and the edges
-denote inheritance. Each edge is annotated with genome coordinates
-defining the specific region(s) inherited.
-(C) This description of inheritance fully defines the
+The Ancestral Recombination Graph defined in terms of genomes embedded
+in a pedigree. (a) Diploid individuals, visualised in a highly inbred pedigree and labelled $i_1$ to $i_8$, contain both paternal (left) and maternal (right) genomes labelled \textsf{A} to \textsf{P}. Black lines show transmission paths connecting genomes in the current generation (\textsf{A} to \textsf{D}) with their ancestors. Two recombination breakpoints, at positions 3 and 6 mean that genomes \textsf{A} and \textsf{C} are independent recombinants of the paternal genomes \textsf{E} and \textsf{F}: the region of genome transmitted is shown in these cases. Where genomes merge into a common ancestor, the merged region is shown darker, until all genomic regions are fully coalesced in genome \textsf{N} (b) The equivalent genome ARG, where there is a single type of node (representing a genome) along with annotations on all edges. Recombination, such that causing the breakpoint at position 3, is associated with two parent genomes (\textsf{E} and  \textsf{F}) and a child genome (the recombinant,  \textsf{A}). In this case two independent recombinations share the same parent genomes, because the individuals containing the recombinants are siblings. Edge-annotation allows this complex genetic relationship to be fully encoded.
```

Comment:
> Recombination, such that causing the breakpoint at 

this doesn't quite parse?

Reply:
I don't think we need the two sentences starting "In this case two .."

Reply:
Typo, should be "such as that"

Reply:
> I don't think we need the two sentences starting "In this case two .."

I was trying to show what a complex relationship looks like.

Happy to delete, but this is the point where we introduce the idea of 2 "recombination nodes" + 1 "recombinant" per recombination, right? The hidden idea here is that in a WF model we can't simply store recombination points on the parents, because parents can be shared. But maybe we just say that in the main text?

Reply:
I think people will get that. Don't want the caption to be too long.

Can you do the line breaks please? Makes it very difficult to comment on specific bits with one long line.

Reply:
Done, thanks.